charts/gha-runner-scale-set-controller/templates/deployment.yaml

```diff
@@ -110,10 +110,16 @@ spec:
         volumeMounts:
         - mountPath: /tmp
           name: tmp
+        {{- range .Values.volumeMounts }}
+        - {{ toJson . }}
```

Comment:
Can you please use toYaml? When someone renders a chart, it would be much easier to read

Reply:
Yes, I can do that. Thanks for the review.